engine/src/mac-window.mm

```diff
@@ -2214,10 +2238,32 @@ bool MCMacDoUpdateRegionCallback(void *p_context, const MCRectangle &p_rect)
 	s_rect_count = 0;
 	MCRegionForEachRect(m_dirty_region, MCMacDoUpdateRegionCallback, m_view);
 	
-	// Force a re-display, this will cause drawRect to be invoked on our view
-	// which in term will result in a redraw window callback being sent.
-	[m_view displayIfNeededInRect: [m_view mapMCRectangleToNSRect:MCRegionGetBoundingBox(m_dirty_region)]];
-	
+	if (MCmajorosversion >= MCOSVersionMake(10,16,0))
+	{
+		// Frequent redraws with displayIfNeeded causes graphical glitches on Macos Big Sur, so instead
+		// we enter the runloop to trigger a redraw. This will cause drawRect to be invoked on our view
+		// which in turn will result in a redraw window callback being sent.
+		// The timeout value of 0.02ms is specified to avoid hitting the 60hz redraw limit.
```

Comment:
This comment is now out of date relative to the constant 2.0 used below (which I believe is seconds?)

Reply:
The edit from 0.02 to 2.0 was a mistake which I have now corrected :/